pkgs/development/tools/pex/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, python3Packages }:
+with python3Packages; buildPythonApplication rec {
+    name = "${pname}-${version}";
```

Comment:
Drop this line

Reply:
Okay, thanks -- is it computed automatically from `pname` and `version`?